doc/Doxyfile.extra.in

```diff
@@ -5,6 +5,5 @@ IMAGE_PATH             = @CMAKE_SOURCE_DIR@/doc/pictures
 FILE_PATTERNS          = *.cc *.cpp *.h *.hh *.hxx
 
 TAGFILES               = \
-"@JRL_MAL_DOXYGENDOCDIR@/jrl-mal.doxytag = @JRL_MAL_DOXYGENDOCDIR@" \
 "@DYNAMIC_GRAPH_DOXYGENDOCDIR@/dynamic-graph.doxytag = @DYNAMIC_GRAPH_DOXYGENDOCDIR@" \
 "@SOT_CORE_DOXYGENDOCDIR@/sot-core.doxytag = @SOT_CORE_DOXYGENDOCDIR@"
```

Comment:
FYI: The JRL CMake module will populate TAGFILES automatically (from this package dependency) if you leave this empty.